i3lock.c

```diff
@@ -2119,6 +2240,53 @@ int main(int argc, char *argv[]) {
 				pass_volume_keys = true;
 				break;
 
+            //custom key commands
+            case 610:
+                hotkeys = true;
+                break;
+            case 620:
+                cmd_brightness_up = strdup(optarg);
+                break;
+            case 621:
+                cmd_brightness_down = strdup(optarg);
+                break;
+
+            case 630:
+                cmd_media_play = strdup(optarg);
```

Comment:
Why is `strdup` necessary here (same with the other uses)?

Reply:
It's the same as the other arguments that take a string input. `strdup` is making a new buffer containing the string. I think that's since the `optarg` buffer isn't necessarily preserved between iterations.

Reply:
I thought it was because `optarg` is a global and mutable string variable. But it isn't, `optarg` seems to be changing its pointer value, so every time we access to optarg we get a different pointer, hence a different string, without modifying previous accesses to optarg. So... it looks like `strdup` is not necessary.

Reply:
`strdup(optarg)` is not made in [`verif_text`](https://github.com/Raymo111/i3lock-color/blob/7668ccea3b4f73806b676a6864faaf5f7406701f/i3lock.c#L1866) and [`wrong_text`](https://github.com/Raymo111/i3lock-color/blob/7668ccea3b4f73806b676a6864faaf5f7406701f/i3lock.c#L1869), yet we have two different values for each one.

Reply:
Ok, I see that. It looks like the only other `strdup(optarg)`s then are for `image_path` and `image_raw_format`.

Reply:
I've made the changes, including to `image_path` and `image_raw_format`, and rebased to remove the conflicts in the completion files.